pkg/cmd/dedicated/register/registercluster.go

```diff
@@ -27,7 +28,8 @@ type options struct {
 	selectedClusterMachinePool    clustersmgmtv1.MachinePool
 	requestedMachinePoolNodeCount int
 	accessKafkasViaPrivateNetwork bool
-	// newMachinePool                clustersmgmtv1.MachinePool
+	pageNumber                    int
```

Comment:
nitpick: Suggesting to use variable names `page` and `size`, since that's what we use for most of the commands.